### PR TITLE
0.40.0-dev0 release prep

### PR DESCRIPTION
**Pull request by Matthew-Jennings**

### CODE REVIEW

**Review by sjswerdloff — COMMENTED** [review]
Need to
Double check the syntax for version info and will do so shortly, but everything else looks fine

**Comment by Matthew-Jennings** [comment]
> Need to Double check the syntax for version info and will do so shortly, but everything else looks fine

Thanks, @sjswerdloff

**Review by sjswerdloff — CHANGES_REQUESTED** [review]
please bump to 0.40.0-dev2 / [0,40,0,"dev2"]

File: lib/pymedphys/_version.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # DO NOT EDIT THIS FILE!
 # This file has been autogenerated by `poetry run pymedphys dev propagate`
 
-version_info = [0, 39, 3]
-__version__ = "0.39.3"
+version_info = [0, 40, 0, "dev0"]
```
Comment (sjswerdloff) [inline lib/pymedphys/_version.py:4]: I think you need to bump to dev2 to be safe...

It appears that there was a 0.40.0-dev1, although I'm not sure if that got packaged and put in to PyPI:
```
commit f8247d17c456cfdd09e1f6bb5b2521b7602e32b5
Author: Simon Biggs <simon.biggs@radiotherapy.ai>
Date:   Thu Oct 6 22:35:33 2022 +1100

    post-release

diff --git a/lib/pymedphys/_version.py b/lib/pymedphys/_version.py
index c9179521..7e1b7833 100644
--- a/lib/pymedphys/_version.py
+++ b/lib/pymedphys/_version.py
@@ -1,5 +1,5 @@
 # DO NOT EDIT THIS FILE!
 # This file has been autogenerated by `poetry run pymedphys dev propagate`
 
-version_info = [0, 39, 2]
-__version__ = "0.39.2"
+version_info = [0, 40, 0, "dev1"]
+__version__ = "0.40.0-dev1"
```

and PyPI definitely already has a 0.40.0-dev0:
Screen Shot 2022-10-30 at 1 58 16 PM

Comment (sjswerdloff) [reply lib/pymedphys/_version.py]: @Matthew-Jennings , given the minimal change, I took the liberty of doing it myself, I hope you don't mind.

**Review by sjswerdloff — APPROVED** [review]

File: lib/pymedphys/_version.py (hunk 1 above)
Comment (Matthew-Jennings) [reply lib/pymedphys/_version.py]: Not at all! Thank you.

**Comment by Matthew-Jennings** [comment]
@sjswerdloff, I notice some checks have been cancelled. Is it safe to merge? Does propagate need re-running after the `dev2` change?

**Comment by sjswerdloff** [comment]
> @sjswerdloff, I notice some checks have been cancelled. Is it safe to merge? Does propagate need re-running after the `dev2` change?

Oh... yes, it does.
Maybe means I shouldn't have edited the file, it should have been done in the pyproject.toml

**Comment by sjswerdloff** [comment]
ugh... I'm making a hash of this.  I think it needs to be done locally so that one can run propagate instead of it happening automatically.

**Comment by Matthew-Jennings** [comment]
> ugh... I'm making a hash of this. I think it needs to be done locally so that one can run propagate instead of it happening automatically.

Yep. Been there before! :(

Let me know if you run out of time and I can finish.

**Comment by sjswerdloff** [comment]
> > ugh... I'm making a hash of this. I think it needs to be done locally so that one can run propagate instead of it happening automatically.
> 
> Yep. Been there before! :(
> 
> Let me know if you run out of time and I can finish.

Looks like it propagated automatically! (yeah @SimonBiggs !)

**Review by SimonBiggs — APPROVED** [review]
LGTM :)

**Pull request merged**